libraries/freertos_plus/standard/tls/src/iot_tls.c

```diff
@@ -730,6 +697,7 @@ BaseType_t TLS_Init( void ** ppvContext,
         pxCtx->xNetworkRecv = pxParams->pxNetworkRecv;
         pxCtx->xNetworkSend = pxParams->pxNetworkSend;
         pxCtx->pvCallerContext = pxParams->pvCallerContext;
+        pxCtx->getDate = prvGetDate_default;
```

Comment:
Where does this come from? Is this left in on accident from the previous change?